applications/RomApplication/custom_strategies/lspg_rom_builder_and_solver.h

```diff
@@ -399,12 +434,21 @@ class LeastSquaresPetrovGalerkinROMBuilderAndSolver : public GlobalROMBuilderAnd
         BuildAndProjectROM(pScheme, rModelPart, A, b, Dx);
 
         // Obtain the assembled residuals vector (To build a basis for Petrov-Galerkin)

```

Comment:
nice addition, although a TODO for a while has been to stop writing to disk

Reply:
I agree, can we take this to a different PR?